### PR TITLE
Add channel option

### PR DESCRIPTION
**Pull request by pizzapim**
Very simple option to specify channel number which defaults to the current default of 0.
I was also thinking of making it a mandatory argument but that would break backwards compatibility, what do you think?

### CODE REVIEW

**Comment by fhunleth** [comment]
The nice part about this library is that there aren't that many users of it. I think we can break backwards compatibility if something looks weird. I'll just bump the version on next release.

File: lib/midi_synth/command.ex [hunk 1]
```diff
@@ -28,27 +28,30 @@ defmodule MIDISynth.Command do
...
+
   @doc """
   Press a key down
   """
-  @spec note_on(note(), velocity()) :: binary()
-  def note_on(note, velocity) do
-    <<0x90, note, velocity>>
+  @spec note_on(note(), velocity(), channel()) :: binary()
+  def note_on(note, velocity, channel) do
+    <<0x90 + channel, note, velocity>>
```
Comment (fhunleth) [inline lib/midi_synth/command.ex]:
```suggestion
    <<0x9::4, channel::4, note, velocity>>
```

This is personal preference to use binary notation to spell out the bits. If nothing else, if channel is out of range, it won't change the command.

Comment (fhunleth) [reply lib/midi_synth/command.ex]: Making `channel` the first parameter feels better to me even though it breaks the API. If it feels better to you, I say we change it. If not, I'm fine with it as it stands too.

File: lib/midi_synth/keyboard.ex [hunk 2]
```diff
@@ -19,11 +19,11 @@ defmodule MIDISynth.Keyboard do
     iex> MIDISynth.Keyboard.play(synth, 60, 100, 80)
     :ok
   """
-  @spec play(GenServer.server(), Command.note(), Command.duration(), Command.velocity()) :: :ok
-  def play(server, note, duration, velocity \\ 127) do
-    MIDISynth.midi(server, Command.note_on(note, velocity))
+  @spec play(GenServer.server(), Command.note(), Command.duration(), Command.velocity(), Command.channel()) :: :ok
+  def play(server, note, duration, velocity \\ 127, channel \\ 0) do
```
Comment (fhunleth) [inline lib/midi_synth/keyboard.ex:23]: Fwiw, I like this order with `channel` being defaulted here. My thought was that people would use `MIDISynth.Keyboard` to check that things worked and maybe for really simple apps, but that it was mostly example code.

**Comment by pizzapim** [comment]
Love that binary notation, really clean! I also added a percussion example to the readme, because that was what I was trying to accomplish with channels :)

**Comment by fhunleth** [comment]
This looks great! Thanks!

Were you planning on sending more changes or shall I go ahead and make a release after work?

**Comment by pizzapim** [comment]
I was going to take a look at some other MIDI stuff like pitch bend and use this library to try it out. Might as well add them to the Command module I guess :). I will probably get to it tomorrow.

**Pull request merged**